### PR TITLE
update Search Default state on mobile

### PR DESCRIPTION
**Pull request by Rohan-cp**
### Summary of Changes

Provide a brief description of the changes made in this PR. Explain the problem you are fixing or the feature you are adding.

### Demo or Before/After Pics

- If this is a new feature, include screenshots or recordings of the feature in action.
- If this PR makes visual changes, include before and after screenshots.

| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2024-04-08 at 20 50 13 | Screenshot 2024-04-09 at 03 34 53 |

### Edge Cases
light mode
Screenshot 2024-04-08 at 20 41 22

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Search/SearchDefault.tsx [hunk 1]
```diff
@@ -81,13 +178,15 @@ export function SearchDefault({ queryRef, blurInputFocus, keyword }: Props) {
...
   return (
     <View className="flex-grow">
-      <FlashList
-        keyboardShouldPersistTaps="always"
-        data={items}
-        estimatedItemSize={25}
-        renderItem={renderItem}
-        onTouchStart={blurInputFocus}
-      />
+      <Suspense fallback={<UserFollowListFallback />}>
```
Comment (Robinnnnn) [inline apps/mobile/src/components/Search/SearchDefault.tsx:181]: place one level higher

**Review by Robinnnnn — APPROVED** [review]
